sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -36,7 +36,26 @@
  */
 public class OpenTelemetryTracer implements com.azure.core.util.tracing.Tracer {
 
-    private final Tracer tracer = GlobalOpenTelemetry.getTracer("Azure-OpenTelemetry");
+    private final Tracer tracer;
+
+    /**
+     * Creates new {@code OpenTelemetryTracer} using default global tracer -
+     * {@code GlobalOpenTelemetry.getTracer()}
+     *
+     */
+    public OpenTelemetryTracer() {
+        this(GlobalOpenTelemetry.getTracer("Azure-OpenTelemetry"));
+    }
+
+    /**
+     * Creates new {@code OpenTelemetryTracer} that wraps {@code io.opentelemetry.api.trace.Tracer}.
+     * Use it for tests.
+     *
+     * @param tracer {@code io.opentelemetry.api.trace.Tracer} implementation.
+     */
+    OpenTelemetryTracer(Tracer tracer) {
```

Comment:
Would there be any case in the future where we'd want to expose this as a public API?

Reply:
there might be:
- let SDKs configure tracer (e.g. for their tests or to provide library name/version)
- use dependency injection to provide tracer instead of static singleton (who knows, maybe in Spring?)

Does it change something? Do you want to make it public right away?

Reply:
Let's hold off on making this public until there is a need to do so. We can keep this as package-private to enable testing for now.